test/api/teaching_period_test.rb

```diff
@@ -28,6 +28,23 @@ def test_get_teaching_periods
     end
   end
 
+  def test_get_teaching_periods_details
+    expected_tp = TeachingPeriod.second
+
+    # perform the GET 
+    get with_auth_token "/api/teaching_periods/#{expected_tp.id}"
```

Comment:
Do you need an auth token to get teaching periods?

Reply:
Hi Andrew, I have adjusted and pushed another commit, please check